### PR TITLE
A11y: Manage Household Default Address Accessibility

### PR DESCRIPTION
**Pull request by andrewyu-salesforce**
# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by andrewyu-salesforce** [comment]
Tracking: `[WI-7954400](https://gus.lightning.force.com/lightning/_classic/%2Fa07B0000008U7nUIAS)`

**Comment by andrewyu-salesforce** [comment]
@force2b @RadSFDeveloper The a11y PR is ready for review

File: src/aura/HH_AddressMgr/HH_AddressMgrHelper.js [hunk 1]
```diff
@@ -10,7 +10,7 @@
         var iAddrSelected;
 
         // figure out if changing default address or creating a new one
-        if (component.get('v.isExistingAddrSectionOpen')) {
+        if (component.get('v.isExistingAddrSectionOpen') == true) {
```
Comment (force2b) [inline src/aura/HH_AddressMgr/HH_AddressMgrHelper.js:13]: Should use `===`

**Review by force2b — COMMENTED** [review]
There are a lot of changes to this, so I'm not really able to tell if they're going to work or not. Overall, I suspect this is going to need a lot of regression testing to make sure that all functionality in ManageHH still work. My memory of this page is that it's quite fragile. Would definitely be nice if these changes help with that fragility of course.

**Comment by andrewyu-salesforce** [comment]
The change is pretty much rewriting the original accordion using SLDS accordion and lighting:button instead of `<div>` with `onclick()`. `onclick()` on `<div>` and `<li>` are not accessible.

**Comment by RadSFDeveloper** [comment]
@rjanjanam , can you please take a look at a failing Manage Household [robot test](https://sfdo-metaci.herokuapp.com/builds/324902/tests) due to change in this PR? Thanks.

**Review by RadSFDeveloper — APPROVED** [review]
Looks good, thanks @andrewyu-salesforce !

**Review by rjanjanam — APPROVED** [review]

**Comment by andrewyu-salesforce** [comment]
@RadSFDeveloper The code is updated and ready for review to keep focus onto the modal and change the role of the existing address button into checkbox. The changes from the past code review is adding keyboard access in the controller file to forced focus and revert the lightning:button back to <div> to add the checkbox role.

**Review by RadSFDeveloper — APPROVED** [review]
Works great, thank you @andrewyu-salesforce !

**Comment by force2b** [comment]
Since I know we want to move this forward I'll take my name off the reviewer list so you're not waiting for me.

**Pull request merged**